news/news-command-doc.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* No news added: instruction for writing news only
```

Comment:
I think this needs a news.  It is important new functionality.  Even if it was mentioned before we want to highlight it.